src/main/kotlin/app/revanced/patches/reddit/misc/disablescreenshotpopup/patch/DisableScreenshotPopupPatch.kt

```diff
@@ -0,0 +1,38 @@
+package app.revanced.patches.reddit.misc.disablescreenshotpopup.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.reddit.misc.disablescreenshotpopup.annotations.DisableScreenshotPopupCompatibility
+import app.revanced.patches.reddit. misc.disablescreenshotpopup.fingerprints.DisableScreenshotPopupFingerprint
+
+@Patch
```

Comment:
Currently, a patch disables the restriction to take a screenshot. Does this patch depend on it? Does Reddit allow taking a screenshot? If not, then this patch should depend on that other patch using `@DependsOn` annotation.

Reply:
no, reddit does not restrict taking screenshot and it does not depend on that patch. this patch simply hides this annoying [popup](https://preview.redd.it/nk3iltxo925b1.png?width=1080&format=png&auto=webp&v=enabled&s=592e66b90ee465ce7894376a2c3aba296c4db04d)